packages/sdk/src/Base.ts

```diff
@@ -163,8 +163,16 @@ class Base {
   }
 
   sendTransaction (transactionRequest: providers.TransactionRequest, chain: TChain) {
-    const chainId = this.toChainModel(chain).chainId
-    return this.signer.sendTransaction({ ...transactionRequest, chainId } as any)
+    chain = this.toChainModel(chain)
+    if (chain.equals(Chain.Polygon)) {
+      if (
+        !transactionRequest.gasLimit ||
+        BigNumber.from(transactionRequest.gasLimit)?.lt(MinPolygonGasLimit)
+      ) {
+        transactionRequest.gasLimit = MinPolygonGasLimit
+      }
+    }
+    return this.signer.sendTransaction({ ...transactionRequest, chainId: chain.chainId } as any)
```

Comment:
@miguelmota how do you feel about setting the gasLimit here? (this change comes from the last commit)

Reply:
I think it made more sense in txOverrides because calling the sdk populate methods uses txOverrides as well

Reply:
I think how you had it last but increase MinPolygonGasLimit to maybe 500k just to be on the safe side